en/Obsidian Publish/Introduction to Obsidian Publish.md

```diff
@@ -0,0 +1,16 @@
+---
+alias: [Obsidian Publish]
+---
+
+Obsidian Publish is a cloud-based service where you can publish your notes to the world without leaving your favorite text editor.
+
+Select the notes you want to share with the world, press 'Publish', and we'll post it to publish.obsidian.md/your-site.
+
+If you're reading this on [help.obsidian.md](https://help.obsidian.md), you're already looking at a site that uses Obsidian Publish. You can see the vault on [GitHub](https://github.com/obsidianmd/obsidian-docs).
+
+## Learn more
+
+- If you a new Obsidian Publish user, learn how to [[Set up Obsidian Publish]].
+- If you have more than one site, learn how to [[Manage sites]].
```

Comment:
maybe add a scenario for people who want to customize how they site looks & work?

Reply:
💯  I'll get on it!